src/types/module.ts

```diff
@@ -14,6 +14,14 @@ export interface ModuleHooks {
 }
 
 export interface ModuleOptions {
+  /**
+   * Set `true` if the project type is Workers.
+   *
+   * If `nitro.experimental.websocket` is enabled, the preset will be set to `cloudflare_durable`, otherwise the preset will be `cloudflare-module`.
```

Comment:
```suggestion
   * If `nitro.experimental.websocket` is enabled, the preset will be set to `cloudflare_durable`, otherwise the preset will be `cloudflare_module`.
```